test/image/mocks/zindex-histogram.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This mock in no longer needed as `histogram` is now tested in another mock. 
Please delete the baseline as well.
Thanks!